Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -97,6 +97,11 @@ def customisePostEra_Run2_2018_pp_on_AA_express_trackingOnly(process):
     customisePostEra_Run2_2018_express_trackingOnly(process)
     return process
 
+def customisePostEra_Run2_2018_cosmics_hybrid(process):
```

Comment:
the name here better match a bit better the scenario name.
So, ```customisePostEra_Run2_2018_cosmics_HI``` here or ```cosmicsEra_Run2_2018_hybrid``` there

Reply:
Thanks for the quick review, I went for `cosmicsEra_Run2_2018_hybrid` because it's not HI after all